File: Lecture/speed/speed.go
package main

import "fmt"

func main() {
	var A [1E+08]uint
	var s uint
	a := &A
	for i := range A {
		a[i] = 2*uint(i) + 1
	}
	for _, e := range a {
		s += e
	}
	fmt.Println(s)
}

// Output : 243.41662ms = 0.243 sec

File: Lecture/speed/speed.py
import numpy as np

def Test():
    A = np.zeros(10**8)
    for i in range(len(A)):
        A[i] = 2*i + 1
    s = 0
    for a in A:
        s += a
    return s

print(Test())

# Output : 37.91
